config/sample_manifest_excel/columns.yml

```diff
@@ -895,6 +895,27 @@ collected_by_for_controls:
   conditional_formattings:
     empty_cell:
     is_error:
+retention_instruction:
+  heading: RETENTION INSTRUCTION
+  updates: retention_instruction
+  unlocked: true
+  validation:
+    options:
+      type: :list
+      formula1: "$A$1:$A$2"
+      allowBlank: false
+      showDropDown: false
```

Comment:
Did you figure out what `showDropDown` does? Just curious as it seems intuitively like it should be `true` for this field.

Reply:
Can we also remove "for the plate " part of the prompt so the column definition can be reused for tubes (e.g. new tube_extraction manifest type) without modification?

prompt: "Please select a retention instruction for the plate from the provided list."

Reply:
showDropDown - Axlsx gem - "A boolean value indicating whether to display a dropdown combo box for a list type data validation." Seems to work fine set to false, I click and can see a dropdown and can select an option.

plate - the whole set of code I've added is plate-specific. The requirement is it only applies only to plate manifests. Maybe we can change that later if they wish to make it useable for tubes. (The vast majority of samples come in on plates, but not all).